src/main/java/frc/robot/subsystems/swerve/factories/constants/RealSwerveConstants.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.swerve.factories.constants;
+
+import com.pathplanner.lib.util.PIDConstants;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.subsystems.swerve.SwerveConstants;
+
+public class RealSwerveConstants {
```

Comment:
Not critical but the name suggests that this extends SwerveConstants and not that it has the Constants as a variable.
Consider changing this class to extend SwerveConstants